package.json

```diff
@@ -25,14 +25,15 @@
   "homepage": "https://github.com/zooniverse/scribes-of-the-cairo-geniza",
   "dependencies": {
     "classnames": "~2.2.5",
-    "counterpart": "^0.18.1",
+    "counterpart": "~0.18.6",
```

Comment:
Is counterpart used anywhere? I couldn't see it.

Reply:
Yes, good call. I forgot to remove this after I decided on `react-localize-redux`.